Hospital.h

```diff
@@ -10,20 +7,20 @@ using namespace std;
 class Hospital {
 private:
     string name;
-    string address;
+    string adress;
```

Comment:
маленькая придирка )))
```suggestion
    string address;
```